src/drivers/gps/params.c

```diff
@@ -86,3 +107,22 @@ PARAM_DEFINE_INT32(GPS_UBX_DYNMODEL, 7);
  */
 PARAM_DEFINE_FLOAT(GPS_YAW_OFFSET, 0.f);
 
+/**
+ * GPS Protocol
+ *
+ * Select the GPS protocol over serial.
+ *
+ * Auto-detection will probe all protocols, and thus is a bit slower.
+ *
+ * @min 0
+ * @max 4
+ * @value 0 Auto detect
+ * @value 1 u-blox
+ * @value 2 MTK
+ * @value 3 Ashtech / Trimble
+ * @value 4 Emlid Reach
+ *
+ * @reboot_required true
+ * @group GPS
+ */
+PARAM_DEFINE_INT32(GPS_PROTOCOL, 1);
```

Comment:
Not per instance?

Reply:
I thought it doesn't need to be, but if you see value in it I can do that.

Reply:
A very small amount of potential value.

Reply:
Alright, let's add it then.